templates/components/itilobject/ticketsatisfaction.html.twig

```diff
@@ -0,0 +1,125 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import "components/form/fields_macros.html.twig" as fields %}
+
+{% block more_fields %}
+    {% if url is defined %}
+        {{ fields.htmlField(
+            '',
+            "<a href='"~ url ~"'>" ~ url ~ "</a><br></div>"
+            ,
+            __('External survey'),
+            {
+                'input_class' : 'col-xxl-9',
+                'label_class' : 'col-xxl-3'
+            }
+        ) }}
+
+    {% else %}
+        <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
+
+        {{ fields.htmlField(
+            '',
+            "<div class='alert alert-info d-flex align-items-center' role='alert'>
+                <div class='text-dark'>
+                    <span class='ms-2'>
+                    <i class='fas fa-xl fa-info-circle'> </i> &nbsp;
+                        " ~ __('After 12 hours, you can no longer modify your notice.') ~ "
+                    </span>
+                </div>
+            </div>"
+            ,
+            '',
+            {
+                'input_class' : 'col-12',
+                'label_class' : ''
+            }
+        ) }}
```

Comment:
Is it possible to write the HTML code without using the `fields.htmlField()` macro?

Reply:
With htmlField, the alert occupies only the width of the form, whereas without htmlField, it occupies the entire width of the page.
![image](https://github.com/glpi-project/glpi/assets/107540223/64bd0e0e-eb63-4c93-a06e-402ce4a8ecfc)

Reply:
assign the html in a variable with `set` directive:

```twig
{% set foo %}
    <div">
        ...
    </div>
{% endset %}
```

and then pass the variable to `htmlField`